javascript/solutions/part1/LongestWordFinder.js

```diff
@@ -0,0 +1,12 @@
+var sentenceString = "This is a long sentence with words of multiple lengths for inputting into your function.";
+var longestWord = '';
+
+//note that this is imperfect for punctuation
```

Comment:
Ha yes. I might list this with others as 'handle more edge cases' in free-form part of problem statement.